test/integration/default/controls/_mapdata.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+control 'TEMPLATE._mapdata' do
```

Comment:
```suggestion
control 'cert._mapdata' do
```

Reply:
:roll_eyes: I `cp` the directory as a dumb.

Thanks a lot.